source/_integrations/switchbot_cloud.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: SwitchBot Cloud
+description: Instructions on how to set up SwitchBot Devices.
+ha_category:
+  - Hub
+  - Plug
+  - Remote
+  - Switch
+ha_release: '2023.10'
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@SeraphicRav'
+ha_domain: switchbot_cloud
+ha_bluetooth: false
+ha_platforms:
+  - switch
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The SwitchBot Cloud integration allows you to control SwitchBot [devices](https://www.switch-bot.com/) connected through the SwitchBot hub. It will automatically discover devices accessible by the hub and remotes configured in the hub using the SwitchBot API.
```

Comment:
```suggestion
The SwitchBot Cloud integration allows you to control SwitchBot [devices](https://www.switch-bot.com/) connected through the SwitchBot hub.
```

I think this is confusing to talk about auto discover as it implies the integration can be auto discovered, but it has to be set up manually.

You can probably add dhcp discovery of the switchbot hub in a future update

https://developers.home-assistant.io/docs/creating_integration_manifest?_highlight=dhcp#dhcp